cmake/DefCmdTarget.cmake

```diff
@@ -0,0 +1,26 @@
+# Defines a target named ${target} and a command with (symbolic) output
+# ${target}-cmd. If ${prg} is undefined the target prints "not found".
+#
+# - Use add_custom_command(…APPEND) to build the command after this.
+# - Use add_custom_target(…DEPENDS) to run the command from a target.
+function(def_cmd_target target prg prg_name prg_fatal)
+  # Define a mostly-empty command, which can be appended-to.
+  add_custom_command(OUTPUT ${target}-cmd
+      WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+      COMMAND ${CMAKE_COMMAND} -E echo "${target}"
+  )
+  # Symbolic (does not generate an artifact).
+  set_source_files_properties(${target}-cmd PROPERTIES SYMBOLIC "true")
+
+  if(prg)
+    add_custom_target(${target}
+      WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+      DEPENDS ${target}-cmd)
+  elseif(prg_fatal)
+    add_custom_target(${target} false
+      COMMENT "${target}: ${prg_name} not found")
```

Comment:
This is fatal now, which means `make lint` will fail if you don't have all the various linters. I think that should be expected. Otherwise your `make lint` will silently fail if some linter is missing.